components/Features/Features.tsx

```diff
@@ -0,0 +1,41 @@
+import Button from './../Button/Button';
+import { Icons } from './constants';
+
+const Features = () => {
+  return (
+    <div className="mx-auto flex max-w-7xl flex-col space-y-12 md:space-y-16">
+      <div className="mx-auto min-w-[343px] max-w-3xl px-4">
+        <p className="bg-violet-700 text-center text-sm font-semibold md:text-base">
+          Integrations
+        </p>
+        <h3 className="mt-3 text-center text-3xl font-semibold text-gray-900 md:text-4xl">
+          Get more value from your tools
+        </h3>
+        <p className="mt-4 text-center  text-lg font-normal text-gray-500 md:mt-5 md:text-xl">
+          Connect your tools, connect your teams. With over 200 apps already
+          available in our directory, your team's favourite tools are just a
+          click away.
+        </p>
+      </div>
+
```

Comment:
Can you split this part?

Reply:
Do you mean to do with "Compound Components" or split with the "<div>" tag?

Reply:
It does not have to be **compound**. I think you can add it as a JSX function too for just splitting list logic from other